minds/minds.py

```diff
@@ -100,9 +99,20 @@ def update(
             f'/projects/{self.project}/minds/{self.name}',
             data=data
         )
+
         if name is not None and name != self.name:
             self.name = name
 
+        refreshed_mind = self.client.minds.get(self.name)
+        self.model_name = refreshed_mind.model_name
```

Comment:
The get method already returns the latest mind: return Mind(self.client, **item) Why do we assign attributes after calling the get API? I don't quite understand this

Reply:
We have to assign the attributes because we are calling update on a minds object which holds state. Once update is called, the state has to be refreshed to the updated one on the same object.
Since update is called on a minds object, returning a new object is not a good practice and can lead to confusions.